readme.md

```diff
@@ -0,0 +1,157 @@
+
+```bash
+$ POSTMARK_SERVER_TOKEN=123 email raw [options]
+
+$ POSTMARK_ACCOUNT_TOKEN=123 servers list [options]
+```
+
+If a token is not supplied as an environment variable, you will be prompted for the server or account token after running the command.
+
+<img src="./media/token-prompt.png" width="100%" style="max-width: 930px;">
+
+## postmark email _&lt;command&gt;_
+
+Send an email via Postmark.
+
+### raw [options]
+
+Send an email with the HTML or Text body defined.
+
+```bash
+$ postmark email raw --from="" --to="" --subject="howdy" --html="<h1>Hi there</h1>" --text="Hi there"
+
+# Options
+--from, -f     Email address you are sending from. Must be an address on a
+                verified domain or confirmed Sender Signature.
+                                                           [string] [required]
+--to, -t       Email address you are sending to            [string] [required]
+--subject      The subject line of the email               [string] [required]
+--html         The HTML version of the email                          [string]
+--text         The text version of the email                          [string]
+```
+
+### template [options]
+
+Send a templated email.
+
+```bash
+$ postmark email template --alias="" --from="" --to="" --model='{"name": "Jane"}'
+
+# Options
+--id, -i     Template ID                                              [string]
+--alias, -a  Template Alias                                           [string]
+--from, -f   Email address you are sending from. Must be an address on a
+             verified domain or confirmed Sender Signature.[string] [required]
+--to, -t     Email address you are sending to              [string] [required]
+--model, -m                                                           [string]
+```
+
+## postmark servers _&lt;command&gt;_
+
+### list [options]
+
+Fetch a list of servers on an account.
+
+```bash
+$ postmark servers list
+
+# Options
+--count, -c   Number of servers to return                             [number]
+--offset, -o  Number of servers to skip                               [number]
+--name, -n    Filter servers by name                                  [string]
+```
+
+## postmark templates _&lt;command&gt;_
+
+Provides a push and pull workflow for your email templates. This lets you store your templates in version control and push to Postmark via a CI/CD pipeline.
+
+**⚠️ Before you get started**, make sure that all of your templates have an alias. Check out our [help doc](https://postmarkapp.com/support/article/1117-how-do-i-use-a-template-alias) for more info.
+
+### pull _&lt;output directory&gt;_ [options]
+
+Download templates from a Postmark server to your local file system.
+
+```bash
+$ postmark templates pull ~/Desktop/my_templates
+
+#Options
+--overwrite, -o  Overwrite templates if they already exist
```

Comment:
I feel like this should be a boolean flag, so if it’s present we set it to true.

For example:
```
postmark templates pull ~/Desktop/templates --overwrite
```

Rather than:

```
postmark templates pull ~/Desktop/templates --overwrite true
```

Reply:
yargs should already interpret this as `true` if it's present in the command.